### PR TITLE
fix minor typo

### PR DESCRIPTION
**Pull request by robitalec**

### CODE REVIEW

**Review by yihui — COMMENTED** [review]
We actually meant:, you should specify metadata in this file, and you should specify metadata only in this file. What would be a better way to express that? Thank you!

File: 12-bookdown.Rmd [hunk 1]
```diff
@@ -57,7 +57,7 @@ As a summary of these files:
 
 - `index.Rmd`: This is the only Rmd document to contain a YAML frontmatter as described within Chapter \@ref(basics), and is the first book chapter.
 
-- Rmd files: A typical **bookdown** book contains multiple chapters, and one chapter lives in one Rmd file.
+- Rmd files: A typical **bookdown** book contains multiple chapters, and each chapter lives in one separate Rmd file.
```
Comment (yihui) [inline 12-bookdown.Rmd:60]: This change makes perfect sense. Thanks!

**Comment by robitalec** [comment]
Right ok - I wasn't sure. Thanks for the clarification. 

I made a couple edits, I hope that they make sense!

**Review by yihui — APPROVED** [review]
Perfect. Thank you!

**Comment by robitalec** [comment]
Thank you @yihui!

**Pull request merged**